addons/weapononback/initSettings.sqf

```diff
@@ -0,0 +1,14 @@
+[
+    QGVAR(enabled), "CHECKBOX",
+    LELSTRING(common,Enabled),
+    LELSTRING(common,ACEKeybindCategoryWeapons),
+    true, false
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(renderLimit), "SLIDER",
+    [LSTRING(RenderLimit), LSTRING(RenderLimitDescription)],
+    LELSTRING(common,ACEKeybindCategoryWeapons),
+    [-1, 100, -1, 0], false,
```

Comment:
```suggestion
    [-1, 100, -1, -1], false,
```
gives only whole numbers, don't need to round

Reply:
@commy2 Missing in documentation.